code/game/objects/structures/syndicate_uplink_beacon.dm

```diff
@@ -0,0 +1,114 @@
+/// Device that traitors can craft in order to be sent a new, undisguised uplink
+/obj/structure/syndicate_uplink_beacon
+	name = "suspicious beacon"
+	icon = 'icons/obj/machines/telecomms.dmi'
+	icon_state = "relay_traitor"
+	desc = "This ramshackle device seems capable of recieving and sending signals for some nefarious purpose."
+	density = TRUE
+	anchored = TRUE
+	/// Traitor's code that they speak into the radio
+	var/uplink_code
```

Comment:
should we type this as a string, like

```suggestion
	var/uplink_code = ""
```

just for complete clarity. keep in mind we can't do `isnull()` stuff this way (because it's not null, it's an empty string) but it's much more clear that it's a string value that has to be assigned at some point during the code

Reply:
makes sense to me! went ahead and did this, thanks